### ISSUE TITLE
Sub tasks indentation

### ISSUE THREAD

**Issue by lyndhurst** [delashum/obsidian-checklist-plugin#25]
I read through the opened and closed issues, and saw you reference the problem of sub tasks not being indented in the task list, and even read one where you imply that you are looking for a way to implement it.
I just wanted to know if that was still in your plans.
Thank you for your time.

**Comment by delashum** [comment]
Yes it is. I have done some work on this, but postponed for a while to work on some lower hanging fruit. I personally also want this, so will definitely get it done soonish

**Comment by lyndhurst** [comment]
Thank you for taking the time to answer, I also believe this would add value to the task list overview.

*I would close the issue with this comment but I thought you might want to leave it visible to avoid duplicates, so I'll leave it up to you.*

**Comment by GamerGirlandCo** [comment]
i'm thinking of opening a PR to close this. however, my approach would introduce dataview as a dependency and i'm not sure how okay @delashum would be with that 😅

**Comment by leandromuto** [comment]
Hi @GamerGirlandCo, can you give more context on why it would need to use DataView?

**Comment by GamerGirlandCo** [comment]
> Hi @GamerGirlandCo, can you give more context on why it would need to use DataView?

[look at the PR](https://github.com/delashum/obsidian-checklist-plugin/pull/161/files#diff-8eb05ff149d5adab63be41fce57779b616a561276add78ecd6a15959ffc17d39) :)

**Comment by leandromuto** [comment]
@GamerGirlandCo I looked at it but I would like to know why you opted to use DataView. I want to know because adding a new plugin dependency will cause performance issues.

**Comment by GamerGirlandCo** [comment]
because 1) it's easier and 2) getting a "nested" data structure for the tasks from indentation alone is fucking difficult. believe me, i've tried...

do you have a better approach or something?

**Comment by sunflowershell** [comment]
> Yes it is. I have done some work on this, but postponed for a while to work on some lower hanging fruit. I personally also want this, so will definitely get it done soonish

Just checking -- did you add a way to do this, or this is still a WIP? 

Thank you.
